pipeline/trusted-resources/README.md

```diff
@@ -2,7 +2,61 @@
 
 [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://github.com/tektoncd/experimental/blob/master/LICENSE)
 
-This is an experimental project to provide a seperate admission webhook for remote resources verification.
+This is an experimental project to provide a seperate webhook for remote resources verification.
+
+- [Install](#install)
+- [Uninstall](#uninstall)
+- [Development](#development)
+
+## Install
+
+Install and configure [`ko`](https://github.com/google/ko).
+
+Install tekton pipeline. To install from source, checkout to pipeline repo and execute:
+```bash
+ko apply -f config/
+```
+
+Then install the new admission webhook:
+```bash
+kubectl create namespace trusted-task
```

Comment:
emmm, this is intentionally. I want to create secret under this ns before `ko apply -f config`. 
And the secret is not so easy to be added to yaml file because it requires me to encode the keys and password.

Do you have better ideas of this?